pkg/test/usersignup/usersignup.go

```diff
@@ -26,20 +26,50 @@ func WithOriginalSub(originalSub string) Modifier {
 	}
 }
 
-func Approved() Modifier {
+// ManuallyApproved sets the UserSignup states to [`approved`]
+func ManuallyApproved() Modifier {
 	return func(userSignup *toolchainv1alpha1.UserSignup) {
-		states.SetApproved(userSignup, true)
+		states.SetManuallyApproved(userSignup, true)
 	}
 }
 
-func ApprovedAutomatically(before time.Duration) Modifier {
+// ManuallyApproved sets the UserSignup state to `approved` and adds a status condition
+func ManuallyApprovedAgo(before time.Duration) Modifier {
```

Comment:
why not `ManuallyApprovedBefore`

Reply:
because the arg is a duration, not a time